backend/controllers/homepage/homepage.ts

```diff
@@ -168,6 +168,13 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
       href: bookAPrisonVisit.ui_url,
       enabled: () => bookAPrisonVisit.ui_url && userHasRoles(['PVB_STAFF']),
     },
+    {
```

Comment:
We normally add a test in `homepageController.test.ts` for each new tile

Reply:
👍 - OK, I've added a unit test in `homepageController.test.ts` and also a cypress test